Vizzuality/heco-invest
# Map uses custom Mapbox' basemap

## Description

This PR adds the custom Mapbox basemap to the map component, as well as updating the frontend `README.md` & `.env.sample` to describe the new env variable for Mapbox's public access token.  

## Testing instructions

Visit `/discover/projects` and verify that:  
- The map works  
- The map uses the new basemap style  

## Tracking

[LET-407](https://vizzuality.atlassian.net/browse/LET-407)

## Screenshot

<img width="1680" alt="Screenshot 2022-06-08 at 12 41 34" src="https://user-images.githubusercontent.com/6273795/172607861-7a57dfe2-56da-4610-b6cf-2974f2a47f00.png">

<img width="986" alt="Screenshot 2022-06-08 at 12 43 36" src="https://user-images.githubusercontent.com/6273795/172608254-3cb6b6c5-e7bd-469b-88f1-ce4b4243d33a.png">
